jobs/RunCwr/config.xml

```diff
@@ -58,7 +58,7 @@ function cleanup_model() {
 }
 trap cleanup_model EXIT
 
-cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
+env MATRIX_OUTPUT_DIR=/srv/artifacts cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
```

Comment:
@kwmonroe I think that I'd like our stack to be as simple as possible when it comes to setting parameters for stuff running up and down the stack. We run the risk of getting focused on what we're doing now, and dropping a lot of CI specific code in cwr, bundletester and matrix, which will make messes for people who aren't doing CI things.

I'd actually rather we moved the --test-id param out of cwr, and just compose and set the results dir on this level, too.